content/src/service/access/AccessCheckerImpl.ts

```diff
@@ -44,9 +44,12 @@ export class AccessCheckerImpl implements AccessChecker {
         return errors
     }
 
+    private SCENE_LOOKBACK_TIME = 5 * 60 * 1000;
```

Comment:
Me have a library called `ms`, so you can do  `SCENE_LOOKBACK_TIME = ms('5m');`

Also, maybe set to static?